tox -e preview fails

Running the preview command has suddenly stopped working for me.  (By "suddenly", I mean "sometime in the last week and I don't think I've done anything silly".)

Here's the output I get:

```
11:30:47 ~/pyconuk/internaldocs $ tox -e preview
preview create: /Users/peteringlesby/pyconuk/internaldocs/.tox/preview
preview installdeps: -r/Users/peteringlesby/pyconuk/internaldocs/requirements.txt
preview installed: alabaster==0.7.7,argh==0.26.1,Babel==2.2.0,docutils==0.12,Jinja2==2.8,livereload==2.4.1,MarkupSafe==0.23,pathtools==0.1.2,Pygments==2.1,pytz==2015.7,PyYAML==3.11,six==1.10.0,snowballstemmer==1.2.1,Sphinx==1.3.1,sphinx-autobuild==0.5.2,sphinx-rtd-theme==0.1.9,tornado==4.3,watchdog==0.8.3,wheel==0.26.0
preview runtests: PYTHONHASHSEED='3886976382'
preview runtests: commands[0] | make preview
sphinx-build -b dirhtml -d build/doctrees   source build/dirhtml
Running Sphinx v1.3.1
loading pickled environment... failed: build environment version not current
building [mo]: targets for 0 po files that are out of date
building [dirhtml]: targets for 5 source files that are out of date
updating environment: 5 added, 0 changed, 0 removed
reading sources... [ 20%] 2015-in-numbers
reading sources... [ 40%] dates
reading sources... [ 60%] index
reading sources... [ 80%] publicity
reading sources... [100%] wash-up-meeting-report

looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [ 20%] 2015-in-numbers
writing output... [ 40%] dates
writing output... [ 60%] index
writing output... [ 80%] publicity
writing output... [100%] wash-up-meeting-report

generating indices... genindex
writing additional pages... search
copying static files... WARNING: html_static_path entry '/Users/peteringlesby/pyconuk/internaldocs/source/_static' does not exist
done
copying extra files... done
dumping search index in English (code: en) ... done
dumping object inventory... done
build succeeded, 1 warning.

Build finished. The HTML pages are in build/dirhtml.
sphinx-autobuild -b dirhtml -d build/doctrees   source build/dirhtml
Traceback (most recent call last):
  File "/Users/peteringlesby/pyconuk/internaldocs/.tox/preview/bin/sphinx-autobuild", line 11, in <module>
    sys.exit(main())
  File "/Users/peteringlesby/pyconuk/internaldocs/.tox/preview/lib/python3.5/site-packages/sphinx_autobuild/__init__.py", line 280, in main
    server.watch(srcdir, builder)
  File "/Users/peteringlesby/pyconuk/internaldocs/.tox/preview/lib/python3.5/site-packages/livereload/server.py", line 203, in watch
    self.watcher.watch(filepath, func, delay, ignore=ignore)
TypeError: watch() got an unexpected keyword argument 'ignore'
make: *** [preview] Error 1
ERROR: InvocationError: '/usr/bin/make preview'
_________________________________________________________________________________________________ summary __________________________________________________________________________________________________
ERROR:   preview: commands failed
```

Here's the version of tox I'm using:

```
11:36:16 ~/pyconuk/internaldocs $ tox --version
2.3.1 imported from /usr/local/lib/python3.5/site-packages/tox/__init__.py
```

This happens with a clean .tox directory.

However, adding `livereload==2.4.0` to requirements.txt still causes tox to use 2.4.1:

```
11:40:45 ~/pyconuk/internaldocs $ git diff
diff --git a/requirements.txt b/requirements.txt
index 1bd61a6..5208a18 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1,3 +1,4 @@
 Sphinx==1.3.1
 sphinx-autobuild==0.5.2
 sphinx_rtd_theme==0.1.9
+livereload==2.4.0
11:42:54 ~/pyconuk/internaldocs $ tox -e preview
preview installed: alabaster==0.7.7,argh==0.26.1,Babel==2.2.0,docutils==0.12,Jinja2==2.8,livereload==2.4.1,MarkupSafe==0.23,pathtools==0.1.2,Pygments==2.1,pytz==2015.7,PyYAML==3.11,six==1.10.0,snowballstemmer==1.2.1,Sphinx==1.3.1,sphinx-autobuild==0.5.2,sphinx-rtd-theme==0.1.9,tornado==4.3,watchdog==0.8.3,wheel==0.26.0
preview runtests: PYTHONHASHSEED='2010381913'
preview runtests: commands[0] | make preview
sphinx-build -b dirhtml -d build/doctrees   source build/dirhtml
Running Sphinx v1.3.1
loading pickled environment... done
building [mo]: targets for 0 po files that are out of date
building [dirhtml]: targets for 0 source files that are out of date
updating environment: 0 added, 0 changed, 0 removed
looking for now-outdated files... none found
no targets are out of date.
build succeeded.

Build finished. The HTML pages are in build/dirhtml.
sphinx-autobuild -b dirhtml -d build/doctrees   source build/dirhtml
Traceback (most recent call last):
  File "/Users/peteringlesby/pyconuk/internaldocs/.tox/preview/bin/sphinx-autobuild", line 11, in <module>
    sys.exit(main())
  File "/Users/peteringlesby/pyconuk/internaldocs/.tox/preview/lib/python3.5/site-packages/sphinx_autobuild/__init__.py", line 280, in main
    server.watch(srcdir, builder)
  File "/Users/peteringlesby/pyconuk/internaldocs/.tox/preview/lib/python3.5/site-packages/livereload/server.py", line 203, in watch
    self.watcher.watch(filepath, func, delay, ignore=ignore)
TypeError: watch() got an unexpected keyword argument 'ignore'
make: *** [preview] Error 1
ERROR: InvocationError: '/usr/bin/make preview'
_________________________________________________________________________________________________ summary __________________________________________________________________________________________________
ERROR:   preview: commands failed
```

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,3 +1,4 @@
 Sphinx==1.3.1
 sphinx-autobuild==0.5.2
 sphinx_rtd_theme==0.1.9
+livereload==2.4.0

